t_modify_http_headers/test_common_config.py

```diff
@@ -228,3 +228,20 @@ def header_not_in(self, header):
 class TestCachedRespSetHeader(TestRespSetHeader):
     cache = True
     directive = "resp_hdr_set"
+
+
+class TestReqHdrSetHost(AddHeaderBase):
+    """
+    Case for `Host` header.
+
+    `Host` header in the request must be replaced with header - "host: host-overriden"
+    """
+
+    cache = False
+    directive = "req_hdr_set"
+
+    def test_req_hdr_set_host(self):
```

Comment:
You should move this test to [test_logic.py](https://github.com/tempesta-tech/tempesta-test/blob/fe2412c63ae7db17a81cbff8647421d614244ee3/t_modify_http_headers/test_logic.py#L103). This test checks the replacement of the `host` header. But `test_common_config.py` contain tests to check Tempesta configuration.

Reply:
The test was moved to `test_logic.py`